docs/content/tutorials/oidc-custom-configuration.md

```diff
@@ -0,0 +1,199 @@
+---
+title: NGINX Ingress Controller and Open Service Mesh
+description: |
+  Use NGINX Ingress Controller with Open Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-1181"
```

Comment:
```suggestion
```
Axing the docs frontmatter as we'll be migrating to a newer system: the current one's being phased out.